scala-days-2018/streams-pgo/TestStream.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Arrays;
+
+public class TestStream {
+
+ static int[] array = new int[] {
+  1,
+  2,
+  3
+ };
+
+ static final int N = 20;
+ static final double EMPLOYMENT_RATIO = 0.5;
+ static final int MAX_AGE = 100;
+ static final int MAX_SALARY = 200_000;
+ /**
+  * @param args the command line arguments
+  */
+ public static void main(String[] args) {
+
+  // Create data set.
+  Person[] persons = new Person[N];
+  for (int k = 0; k < N; ++k) {
+   persons[k] = new Person(Math.random() > EMPLOYMENT_RATIO ? Employment.EMPLOYED : Employment.UNEMPLOYED, (int)(Math.random() * MAX_SALARY), (int)(Math.random() * MAX_AGE));
+  }
+  long sum = 0;
+  for (int j = 1; j < 21; j++) {
+   long time = System.currentTimeMillis();
+   for (int i = 0; i < 1000000; ++i) {
+    getValue(persons);
+   }
+   long currentTime = (System.currentTimeMillis() - time);
+   sum += currentTime;
+   System.out.println("Iteration " + j + " finished in " + currentTime + " milliseconds.");
+  }
+  System.out.println("TOTAL time: " + sum);
+ }
+
+ public static double getValue2() {
```

Comment:
What is this code for? Seems unused and also not mentioned in the readme (same for the non-PGO file).

Do we really want the same file committed twice? Why not just have two .md files in the same directories to explain the non-PGO and the PGO part?

Reply:
This must be testing code that I left by accident. I'll fix this in the morning.